cmd/cmd.go

```diff
@@ -42,6 +43,8 @@ const (
 	FlagStatusAddr = "status-addr"
 	// FlagSlowLogFile is the name of slow-log-file flag.
 	FlagSlowLogFile = "slow-log-file"
+	// FlagRedactInfoLog is whether to redact sensitive information in log.
+	FlagRedactInfoLog = "redact-info-log"
```

Comment:
What about warning/error/debug log?

Reply:
They will also be redacted. I use this name to keep consistent with `tikv` and `pd`.

Reply:
I prefer just calling it `--redact-log` following TiDB's session var.

TiKV uses the word "info-log" to distinguish from other log-like stuff such as WAL. I don't know what's going on in PD. Also here it is a CLI parameter not a config.